receiver/jmxreceiver/config.go

```diff
@@ -11,55 +11,81 @@ import (
+
 type Config struct {
-	// The path for the JMX Metric Gatherer uber JAR (/opt/opentelemetry-java-contrib-jmx-metrics.jar by default).
+	// The path for the JMX Metric Gatherer or JMX Scraper JAR (/opt/opentelemetry-java-contrib-jmx-metrics.jar by default).
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	JARPath string `mapstructure:"jar_path"`
 	// The Service URL or host:port for the target coerced to one of form: service:jmx:rmi:///jndi/rmi://<host>:<port>/jmxrmi.
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	Endpoint string `mapstructure:"endpoint"`
 	// The target system for the metric gatherer whose built in groovy script to run.
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	TargetSystem string `mapstructure:"target_system"`
 	// The duration in between groovy script invocations and metric exports (10 seconds by default).
 	// Will be converted to milliseconds.
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	CollectionInterval time.Duration `mapstructure:"collection_interval"`
-	// The exporter settings for
+	// The OTLP exporter settings
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	OTLPExporterConfig otlpExporterConfig `mapstructure:"otlp"`
 	// The JMX username
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	Username string `mapstructure:"username"`
 	// The JMX password
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	Password configopaque.String `mapstructure:"password"`
 	// The keystore path for SSL
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	KeystorePath string `mapstructure:"keystore_path"`
 	// The keystore password for SSL
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	KeystorePassword configopaque.String `mapstructure:"keystore_password"`
 	// The keystore type for SSL
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	KeystoreType string `mapstructure:"keystore_type"`
 	// The truststore path for SSL
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	TruststorePath string `mapstructure:"truststore_path"`
 	// The truststore password for SSL
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	TruststorePassword configopaque.String `mapstructure:"truststore_password"`
 	// The truststore type for SSL
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	TruststoreType string `mapstructure:"truststore_type"`
 	// The JMX remote profile.  Should be one of:
 	// `"SASL/PLAIN"`, `"SASL/DIGEST-MD5"`, `"SASL/CRAM-MD5"`, `"TLS SASL/PLAIN"`, `"TLS SASL/DIGEST-MD5"`, or
 	// `"TLS SASL/CRAM-MD5"`, though no enforcement is applied.
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	RemoteProfile string `mapstructure:"remote_profile"`
 	// The SASL/DIGEST-MD5 realm
+	// Supported by: jmx-scraper and jmx-metric-gatherer
 	Realm string `mapstructure:"realm"`
 	// Array of additional JARs to be added to the class path when launching the JMX Metric Gatherer JAR
+	// Supported by: jmx-scraper and jmx-metric-gatherer
```

Comment:
When using additional jars, the command to start jmx-scraper is different, an example is provided [here](https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/jmx-scraper/README.md#extra-libraries-in-classpath).